zds/settings.py

```diff
@@ -147,7 +146,7 @@
                 'zds.utils.context_processor.git_version',
             ],
             'debug': DEBUG,
-            }
+        }
```

Comment:
`Flake8 Error E123 — closing bracket does not match indentation of opening bracket's lineat line 150 col 13`